phpBB/phpbb/storage/storage.php

```diff
@@ -229,4 +336,66 @@ public function get_link($path)
 	{
 		return $this->get_adapter()->get_link($path);
 	}
+
+	/**
+	 * Get total storage size.
```

Comment:
In other comments as well :)
I don't think we use full stops in other files so using it seems a bit out of place